Editor/Scripts/MeshSyncServerInspector.cs

```diff
@@ -85,6 +155,44 @@ public void DrawServerSettings(MeshSyncServer t)
         }
     }
 
+    private void DrawInstanceSettings(MeshSyncServer t)
+    {
+        var style = EditorStyles.foldout;
+        style.fontStyle = FontStyle.Bold;
+        t.foldInstanceSettings = EditorGUILayout.Foldout(t.foldInstanceSettings, "Instances", true, style);
+        if (t.foldInstanceSettings)
+        {
+            t.InstanceHandling = (BaseMeshSync.InstanceHandlingType)EditorGUILayout.EnumPopup("Instance handling", t.InstanceHandling);
+
+            DrawPrefabListElement(t);
+        }
+
+        EditorGUILayout.LabelField($"Instance count: {t.InstanceCount}");
+    }
+
+    static void DrawPrefabListElement(MeshSyncServer t)
+    {
+        if (t.prefabList.Count > 0)
+        {
+            EditorGUILayout.LabelField("Instance prefabs:");
+
+            foreach (var prefabHolder in t.prefabList)
+            {
+                Rect rect = EditorGUILayout.BeginHorizontal();
+
+                EditorGUILayout.LabelField(prefabHolder.name);
+
+                EditorGUILayout.ObjectField(prefabHolder.prefab, typeof(GameObject), true);
```

Comment:
there is a `EditorGUILayout.ObjectField()` function which receives a label, if that's what is intended here.

Reply:
Thanks, fixed